bin/install/install-dcos-cli.sh

```diff
@@ -4,7 +4,44 @@ set -o errexit -o pipefail
 
 usage()
 { # Show usage information.
-  echo "$(basename "$(test -L "$0" && readlink "$0" || echo "$0")") <installation-path> <dcos-url>"
+  echo "$(basename "$(test -L "$0" && readlink "$0" || echo "$0")") <installation-path> <dcos-url> [--add-path yes/no]"
+}
+
+post_install_message()
+{
+    echo 'Finished installing and configuring DCOS CLI.'
+    echo ''
+    echo 'Run this command to set up your environment and to get started:'
+    echo "source $1 && dcos help"
+}
+
+RC_NAME=""
+
+add_dcos_path_to_profile()
+{
+    UNAME=`uname`
+    case "$UNAME" in
+        Linux ) RC_NAME=".bashrc";;
+        Darwin ) RC_NAME=".bash_profile";;
+        CYGWIN* ) RC_NAME=".bashrc";;
+        MINGW* ) RC_NAME=".profile";;
+        * ) RC_NAME=".bashrc";;
+    esac
+    echo "" >> ~/"$RC_NAME"
+    echo "# add the path to the DCOS binary to your bash profile" >> ~/"$RC_NAME"
+    echo "export PATH=$1:\$PATH" >> ~/"$RC_NAME"
```

Comment:
We should be adding the CLI to the end of $PATH to maintain precedence of the system Python (otherwise we'll use the virtualenv Python first when the user types in `python`)